piaxe/miner.py

```diff
@@ -37,6 +37,7 @@
 from . import bm1366
 from . import influx
 from . import discord
+from .ssd1306 import SSD1306
```

Comment:
please add a try except block or move it into the other try except block that imports raspberry specific libraries